client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import ClipboardButtonInput from 'components/clipboard-button-input';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import QueryPluginKeys from 'components/data/query-plugin-keys';
+import { getPurchase, isDataLoading } from '../../utils';
```

Comment:
I'd avoid relative paths at all costs, especially such with 2 directories above - they often break as we move files here and there. And `client/me/purchases/utils` isn't that long anyway 😉